lib/Github/Api/Repository/Assets.php

```diff
@@ -36,10 +39,14 @@ public function all($username, $repository, $id)
      * @param string $repository the name of the repo
      * @param int    $id         the id of the asset
      *
-     * @return array
+     * @return array|string
      */
-    public function show($username, $repository, $id)
+    public function show($username, $repository, $id, bool $returnBinaryContent = false)
```

Comment:
missing phpdoc :)

Reply:
I didn't add it because it's fully documented in the typehint